pandas/core/groupby/groupby.py

```diff
@@ -2235,8 +2235,12 @@ def _get_cythonized_result(
         grouper = self.grouper
 
         labels, _, ngroups = grouper.group_info
-        output: Dict[base.OutputKey, np.ndarray] = {}
+        output: Dict[base.OutputKey, np.ndarray, str:str] = {}
         base_func = getattr(libgroupby, how)
+        from pandas.core.groupby.generic import DataFrameGroupBy
```

Comment:
Is there a way to get the name without this runtime import?

Reply:
I looked into that already, but could not find any other way. SeriesGroupBy and DataFrameGroupBy have no matching attribute, which would allow us to decide, if it is the former or the latter.

I found a similar run time import in line 2540, so I though that would be a possible solution.

Reply:
I think you can call `self._selected_obj` to get the underlying `Series` or `DataFrame` and then try to `getattr` the `column.name`. Preferable to avoid the runtime import where possible.

Reply:
Thanks for the hint. I was able to avoid the runtime import this way.